rel/overlay/etc/default.ini

```diff
@@ -47,7 +47,7 @@ changes_doc_ids_optimization_threshold = 100
 max_document_size = 8000000 ; bytes
 ;
 ; Maximum attachment size.
-; max_attachment_size = infinity
+; max_attachment_size = 1073741824 ; 1 gibibyte
```

Comment:
I'd go even lower tbh but anything is an improvement over unlimited.

Reply:
I support decreasing this over time. I wouldn't want to change this number right now, because I believe that would change the scope of the ongoing vote on the ML.